develop/TOOLS/TOONuGet/Producing_NuGet.md

```diff
@@ -4,80 +4,153 @@ uid: Producing_NuGet
 
 # Producing NuGet packages
 
+> [!IMPORTANT]
+> This section might include some information that is only applicable to Skyline employees and/or links that are only accessible to Skyline employees.
+
 Automatic creation and publishing of NuGets from the community class library or from [custom solutions](xref:Custom_solution_development_with_CI_CD_Pipeline) is supported through the Skyline CI/CD pipelines.
 
 On the [custom solutions tab](xref:Repository_types#custom-solutions) of the SLC SE RepoManager tool, you can enable the NuGet-related stages by clicking the NuGet button. This will add a *JenkinsNuGetConfiguration.xml* file that allows configuration of these stages (enable/disable creation, signing, publishing of NuGet packages). For more information on the expected content of this file, refer to the [NuGet stages XML schema](xref:SchemaNuGetStagesConfig) documentation.
 
-> [!IMPORTANT]
->
-> - The NuGet name must always begin with `Skyline.DataMiner.`.
-> - The NuGet Name must be unique and limited to 50 characters.
+The pipeline will automatically create pre-release packages, which can be used during development. Every build will have its own pre-release package.
+Every release build will have a signed released package.
+
+## General
+
+Below rules are mainly focused for NuGets that are published to nuget.org. It is highly adviced to use the same rules for internal NuGets as well as that would make it easy transferable to nuget.org when necessary.
+
+### Mandatory rules for nuget.org
 
-To configure the name and version of your NuGet packages, all standard options are available. The easiest way to do this is to go to the project settings and adjust the assembly info.
+- NuGet needs to include a license file
+- NuGet needs to be signed as it will not be accepted
```

Comment:
NuGet needs to be signed as it will not be accepted => NuGet needs to be signed as only signed packages can be published on nuget.org